test/dummy/db/schema.rb

```diff
@@ -1512,6 +1512,7 @@
     t.string   "texto"
     t.integer  "tag_id"
     t.integer  "subtipo_id"
+    t.integer  "suscripciones_count"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.